### PR TITLE
History file generation in dataset evaluation process

### PR DESCRIPTION
**Pull request by gentlecat**
:scroll: :honeybee: :honeybee: :honeybee:

### CODE REVIEW

**Comment by gentlecat** [comment]
Still need to permanently save that file on the server and provide a way to download it.

File: dataset_eval/evaluate.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import re
 
 SLEEP_DURATION = 30  # number of seconds to wait between runs
+HISTORY_STORAGE_DIR = os.path.join(config.FILE_STORAGE_DIR, "history")
```
Comment (alastair) [inline dataset_eval/evaluate.py]: does it make sense to put the existing high level files here too? what do you think?

**Comment by alastair** [comment]
looks good :rocket:

File: dataset_eval/evaluate.py (hunk 1 above)
Comment (gentlecat) [inline dataset_eval/evaluate.py]: What files?

Comment (alastair) [inline dataset_eval/evaluate.py]: I mean all of the ones that we currently use in the high level extractor currently.

Comment (gentlecat) [inline dataset_eval/evaluate.py]: *http://chatlogs.musicbrainz.org/musicbrainz-devel/2015/2015-08/2015-08-15.html#T12-14-36-810697* :memo:

**Pull request merged**